src/index.scss

```diff
@@ -0,0 +1,437 @@
+*,
```

Comment:
This is a small project so it is not causing any real issues but generally you should not place all styles in one big SCSS file. I would suggest using for example CSS modules which should work out of the box in CRA applications.